saleor/graphql/checkout/mutations/checkout_remove_promo_code.py

```diff
@@ -84,31 +84,28 @@ def perform_mutation(
             "promo_code", promo_code, "promo_code_id", promo_code_id
         )
 
-        object_type, promo_code_pk = cls.clean_promo_code_id(promo_code_id)
-
         checkout = get_checkout(cls, info, checkout_id=checkout_id, token=token, id=id)
 
         manager = get_plugin_manager_promise(info.context).get()
         checkout_info = fetch_checkout_info(checkout, [], manager)
 
-        removed = False
         if promo_code:
-            removed = remove_promo_code_from_checkout(checkout_info, promo_code)
+            remove_promo_code_from_checkout_or_error(checkout_info, promo_code)
```

Comment:
You should add `try, except` here and add a info about invalid field, like:
```suggestion
try:
    remove_promo_code_from_checkout_or_error(checkout_info, promo_code)
except ValidationError as error:
    raise ValidationError({
        "promo_code": error
    })
```